opent2t-reference/main.js

```diff
@@ -105,6 +109,12 @@ app.loadConfigs = function () {
     return deferred.promise;
 }
 
+app.getKnownHubs = function () {
+    let hubTranslators = glob.sync('./node_modules/opent2t-translator-com-*-hub');
```

Comment:
Quick question: Do you add the hub's in as dependencies for this reference app package? Or do we install these manually ?